runtime/doc/lua51refvim.txt

```diff
@@ -0,0 +1,5617 @@
+*lua51refvim.txt*        Lua 5.1 Reference Manual for Vim
+                                Vim version 7.0
+
+
+                                                      *lua51ref* *Lua51-Reference*
+                             Lua Reference for Vim
+                           =========================
+                                 Version 0.2.0
+                              November, 25th, 2006
+
+                           (c) 2006 by Luis Carvalho
+                         <lexcarvalho at gmail dot com>
+
+                    Adapted from "Lua: 5.1 reference manual"
+                 R. Ierusalimschy, L. H. de Figueiredo, W. Celes
+                      Copyright (c) 2006 Lua.org, PUC-Rio.
+
+
+                 See |lrv51-doc| for information on this manual.
+                 See |lrv51-copyright| for copyright and licenses.
+
+
+                                   CONTENTS
+                                 ============
+
+           1   INTRODUCTION........................|lrv51-intro|
```

Comment:
we probably change the prefix to something less keysmashy, maybe `luaref`, which also can be the filename. ("LuaRef" means something else in the C code base, but this term is not leaked to public docs, so there is no confusion)